scapy/as_resolvers.py

```diff
@@ -91,23 +91,31 @@ def resolve(self, *ips):
             asn,ip,desc = map(str.strip, l.split("|"))
             if asn == "NA":
                 continue
-            asn = int(asn)
-            ASNlist.append((ip,asn,desc))
+            asn = "AS" + str(int(asn))
```

Comment:
I wanted to make all functions return the same thing:
`AS_resolver_radb` and `AS_resolver_riswhois` return `AS[number]`